code/game/objects/items/robot/robot_upgrades.dm

```diff
@@ -292,3 +292,47 @@
 			msg_cooldown = world.time
 	else
 		deactivate()
+
+
+
+
+///AI Upgrades

```

Comment:
Honest, I do not believe these belong here. This file is for **cyborg** upgrades, not AIs.